TribalWars/src/Infrastructure/Migrations/20210430085658_add_attackVillage.Designer.cs

```diff
@@ -0,0 +1,1158 @@
+﻿// <auto-generated />
+using System;
+using Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Infrastructure.Migrations
+{
+    [DbContext(typeof(ApplicationDbContext))]
+    [Migration("20210430085658_add_attackVillage")]
```

Comment:
Try to have consistency in migration naming. For example - don't mix snake_case with camelCase. Name this migration like this 'add_attack_village'.
You have to change Migration() attribute and file name.